x/stake/client/cli/utils.go

```diff
@@ -0,0 +1,90 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authcmd "github.com/cosmos/cosmos-sdk/x/auth/client/cli"
+	"github.com/cosmos/cosmos-sdk/x/stake"
+	"github.com/cosmos/cosmos-sdk/x/stake/types"
+	"github.com/pkg/errors"
+)
+
+// nolint: gocyclo
+// TODO: Make this pass gocyclo linting
```

Comment:
gocyclo is now removed, so we can probably adapt this TODO to just be to reduce complexity

Reply:
👍